poradnia/stats/tests.py

```diff
@@ -23,6 +24,14 @@ def polyfill_http_response_json():
         setattr(HttpResponse, 'json', lambda x: json.loads(x.content))
 
 
+def purge_users(func):
+    def func_wrapper(*args, **kwargs):
+        for user in User.objects.all():
```

Comment:
- Faktycznie, lepiej będzie to wyglądało z `User.objects.all().delete()`.
- Myślę, że lepiej będzie zostać przy usuwaniu wszystkich użytkowników zamiast jednego - celem dekoratora jest właśnie całkowite pozbycie się automatycznie stworzonych użytkowników, niezależnie od tego ilu ich faktycznie jest (0 lub więcej). W tym przypadku oczywiście zawsze mamy tylko jednego użytkownika, ale myślę, że usuwanie wszystkich lepiej oddaje intencje